app/scripts/templates/complete_account_unlock.mustache

```diff
@@ -0,0 +1,37 @@
+<div id="main-content">
+  {{#isLinkExpired}}
```

Comment:
incorrect, this path occurs if the user deletes the account then opens an old unlock email.